pymc_experimental/tests/distributions/test_continuous.py

```diff
@@ -159,3 +159,26 @@ def test_logcdf(self):
             {"nu": Rplus},
             lambda value, nu: sp.chi.logcdf(value, df=nu),
         )
+
+
+class TestMaxwell:
+    """
+    Wrapper class so that tests of experimental additions can be dropped into
+    PyMC directly on adoption.
+    """
+
+    def test_logp(self):
+        check_logp(
+            Maxwell,
+            Rplus,
+            {"a": Rplus},
```

Comment:
bit confused why the error is coming up

```text
Failed: test_params={'a': -1.0, 'value': array(0.01)}
```

Reply:
seeing that these are the `skip_paramdomain_outside_edge_test` section of the `check_logp` and `check_logcdf` tests.

Reply:
I think this may be fine. Negative scales will just flip the distribution along the y axis. Maybe just mention that in the docstrings.

Reply:
I went with the check in order to stick with the typical maxwell formulation and test against that